GenerateIssuerIconAssets.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+command -v ag >/dev/null 2>&1 || { echo >&2 "I require The Silver Searcher (ag) but it's not installed. Aborting."; exit 1; }
+command -v sips >/dev/null 2>&1 || { echo >&2 "I require sips but it's not installed. Aborting."; exit 1; }
+
+get_name() {
+  # RegEx Explanation:
+  #   Positive Lookbehind -- (?<=\.\/IssuerIcons\/)
+  #     Asserts that the selected portion must be preceeded by ./IssuerIcons/
+  #   Pattern Match -- .*
+  #     Matches all characters, excluding newlines, of any length
+  #   Positive Lookforward -- (?=\.png)
+  #     Asserts that the selected portion must be followed by .png
+  echo $1 | ag --only-matching '(?<=\.\/IssuerIcons\/).*(?=\.png)'
+}
+
+write_json() {
+  # JSON copied from Xcode output
+  echo -n "{
```

Comment:
Using a heredoc would make this cleaner:
```
cat << EOF
{
  "example": "$1"
}
EOF
```
You won't have to escape the quotes, and you still get variable expansion!

To redirect the output:
```
cat << EOF > output
...
EOF
```